connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -199,6 +200,17 @@ message AnalyzePlanRequest {
     // (Required) The logical plan to get the storage level.
     Relation relation = 1;
   }
+
+  message Checkpoint {
+    // (Required) The logical plan to checkpoint.
+    Relation relation = 1;
+
+    // (Optional) Is it localCheckpoint
```

Comment:
Can the documentation be a bit less anaemic? Tell the world what this means. The same applies for the `eager` field.